tools/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
-FROM shaomai/craftassist:latest as base
+FROM shaomai/craftassist:1 as base
```

Comment:
check-in the changes to this docker image also within this PR

Reply:
Dockerfile.base is the one we use to build this docker image and the changes have already been checked in. Since you asked, I feel that the naming is confusing so moved `tools/docker/Dockerfile.base` to a subfolder `tools/docker/craftassist_docker/Dockerfile` and added a README to clarify, thanks!